app/services/discussion_service.rb

```diff
@@ -79,6 +79,11 @@ def self.mark_as_read(discussion:, params:, actor:)
     DiscussionReader.for(user: actor, discussion: discussion).viewed! target_to_read.created_at
   end
 
+  def self.dismiss(discussion:, params:, actor:)
+    actor.ability.authorize! :show, discussion
```

Comment:
infact.. I had to add dismiss to ability for load_and_authorize.. which was good. I think I was just running on autopilot.. Thank goodness for CR